### PR TITLE
Trivial typo fix in Streams docs

### PR DESCRIPTION
**Pull request by tkopczynski**

### CODE REVIEW

**Comment by johanandren** [comment]
I can't figure out what the change is and github marks the entire section as changed even though it looks identical?

File: akka-docs/src/main/paradox/general/stream/stream-design.md [hunk 1]
```diff
@@ -65,7 +65,7 @@ Another not obvious gain from hiding the Reactive Streams interfaces comes from
...
+The other, and perhaps more important reason for hiding the Reactive Streams interfaces comes back to the first points of this explanation: the fact of Reactive Streams being an SPI, and as such is hard to "get right" in ad-hoc implementations. Thus Akka Streams discourages, the use of the hard to implement pieces of the underlying infrastructure, and offers simpler, more type-safe, yet more powerful abstractions for users to work with: GraphStages and operators. It is of course still (and easily) possible to accept or obtain Reactive Streams (or JDK+ Flow) representations of the stream operators by using methods like `asPublisher` or `fromSubscriber`.
```
Comment (helena) [inline akka-docs/src/main/paradox/general/stream/stream-design.md]: So the typo is `streams` => `Streams`. While you are in there, perhaps remove the comma after `discourages`? I'm not sure why it was in there.

```suggestion
The other, and perhaps more important reason for hiding the Reactive Streams interfaces comes back to the first points of this explanation: the fact of Reactive Streams being an SPI, and as such is hard to "get right" in ad-hoc implementations. Thus Akka Streams discourages the use of the hard to implement pieces of the underlying infrastructure, and offers simpler, more type-safe, yet more powerful abstractions for users to work with: GraphStages and operators. It is of course still (and easily) possible to accept or obtain Reactive Streams (or JDK+ Flow) representations of the stream operators by using methods like `asPublisher` or `fromSubscriber`.
```

**Comment by patriknw** [comment]
OK TO TEST

**Review by patriknw — APPROVED** [review]
LGTM

**Review by helena — APPROVED** [review]
LGTM

**Pull request merged**